src/store/workers/workersSlice.ts

```diff
@@ -288,33 +288,29 @@ export const handleHostCall = createAsyncThunk("workers/handleHostCall", async (
     }
 
     await Promise.all(
-      state.workers
-        // [KF] Remove or change this condition when we support host calls on more PVMs.
-        .filter(({ id }) => id === "typeberry")
-        .map(async (worker) => {
-          const resp = await asyncWorkerPostMessage(worker.id, worker.worker, {
-            command: Commands.HOST_CALL,
-            payload: { hostCallIdentifier: worker.exitArg as HostCallIdentifiers },
-          });
-
-          if (
-            resp.payload.hostCallIdentifier === HostCallIdentifiers.WRITE &&
-            resp.payload.storage &&
-            // Remove if we decide to make storage initialization optional
-            state.debugger.storage
-          ) {
-            const newStorage = mergePVMAndDebuggerEcalliStorage(resp.payload.storage, state.debugger.storage);
-            dispatch(setStorage(newStorage));
-          }
+      state.workers.map(async (worker) => {
+        const resp = await asyncWorkerPostMessage(worker.id, worker.worker, {
+          command: Commands.HOST_CALL,
+          payload: { hostCallIdentifier: worker.exitArg as HostCallIdentifiers },
+        });
+        if (
+          resp.payload.hostCallIdentifier === HostCallIdentifiers.WRITE &&
```

Comment:
These `HostCallIndentifiers` should rather come from host calls impls (`new Write().index`) to avoid duplicating this stuff in case it ever changes.